### PR TITLE
Documentation fixes

### PR DESCRIPTION
**Pull request by MarPetra**
Updates the references to old documentation

### CODE REVIEW

File: doc/bdDijkstra/bdDijkstra-family.rst [hunk 1]
```diff
@@ -12,8 +12,8 @@
 .. rubric:: Previous versions of this page
 
 * **Supported versions:**
-  `Latest <https://docs.pgrouting.org/latest/en/pgrbdDijkstra.html>`__
-  (`3.2 <https://docs.pgrouting.org/3.2/en/pgr_bdDijkstra.html>`__)
+  `Latest <https://docs.pgrouting.org/latest/en/pgrbdDijkstra-family.html>`__
+  (`3.2 <https://docs.pgrouting.org/3.2/en/pgr_bdDijkstra-family.html>`__)
```
Comment (cvvergara) [inline doc/bdDijkstra/bdDijkstra-family.rst:16]: Line 15 y 16 remove the `pgr_` from `pgr_bdDijkstra-family.html`

**Review by cvvergara — CHANGES_REQUESTED** [review]

File: doc/bdDijkstra/pgr_bdDijkstraCost.rst [hunk 2]
```diff
@@ -7,6 +7,17 @@
     Alike 3.0 License: https://creativecommons.org/licenses/by-sa/3.0/
    ****************************************************************************
 
+|
+
+* **Supported versions:**
+  `Latest <https://docs.pgrouting.org/latest/en/pgr_dijkstraCost.html>`__
+  (`3.2 <https://docs.pgrouting.org/3.2/en/pgr_dijkstraCost.html>`__)
```
Comment (cvvergara) [inline doc/bdDijkstra/pgr_bdDijkstraCost.rst:14]: The html file names should match the one on 3.1

**Review by cvvergara — CHANGES_REQUESTED** [review]

File: doc/breadthFirstSearch/pgr_binaryBreadthFirstSearch.rst [hunk 3]
```diff
@@ -7,6 +7,11 @@
     Alike 3.0 License: http://creativecommons.org/licenses/by-sa/3.0/
    ****************************************************************************
 
+|
+
+* **Supported versions:**
+  `Latest <https://docs.pgrouting.org/latest/en/pgr_binaryBreadthFirstSearch.html>`__
```
Comment (cvvergara) [inline doc/breadthFirstSearch/pgr_binaryBreadthFirstSearch.rst:13]: Add also:
```
`(3.2 <https://docs.pgrouting.org/3.2/en/pgr_binaryBreadthFirstSearch.html>`__)
```

File: doc/breadthFirstSearch/pgr_breadthFirstSearch.rst [hunk 4]
```diff
@@ -7,6 +7,11 @@
     Alike 3.0 License: http://creativecommons.org/licenses/by-sa/3.0/
    ****************************************************************************
 
+|
+
+* **Supported versions:**
+  `Latest <https://docs.pgrouting.org/latest/en/pgr_breadthFirstSearch.html>`__
```
Comment (cvvergara) [inline doc/breadthFirstSearch/pgr_breadthFirstSearch.rst:13]: Add also:
```
`(3.2 <https://docs.pgrouting.org/3.2/en/pgr_breadthFirstSearch.html>`__)
```

File: doc/chinese/chinesePostmanProblem-family.rst [hunk 5]
```diff
@@ -7,6 +7,14 @@
     Alike 3.0 License: https://creativecommons.org/licenses/by-sa/3.0/
    ****************************************************************************
 
+|
+
+* **Supported versions:**
+  `Latest <https://docs.pgrouting.org/latest/en/chinesePostmanProblem.html>`__
+  (`3.2 <https://docs.pgrouting.org/3.2/en/pgr_chinesePostman.html>`__)
```
Comment (cvvergara) [inline doc/chinese/chinesePostmanProblem-family.rst:14]: Add the `-family`
```
  `Latest <https://docs.pgrouting.org/latest/en/chinesePostmanProblem-family.html>`__
  (`3.2 <https://docs.pgrouting.org/3.2/en/pgr_chinesePostman-family.html>`__)
```

**Review by cvvergara — CHANGES_REQUESTED** [review]

File: doc/components/components-family.rst [hunk 6]
```diff
@@ -7,6 +7,17 @@
     Alike 3.0 License: https://creativecommons.org/licenses/by-sa/3.0/
    ****************************************************************************
 
+|
+
+* **Supported versions:**
+  `Latest <https://docs.pgrouting.org/latest/en/components.html>`__
+  (`3.2 <https://docs.pgrouting.org/3.2/en/components.html>`__)
```
Comment (cvvergara) [inline doc/components/components-family.rst:14]: The name is `components-family`

File: doc/components/pgr_makeConnected.rst [hunk 7]
```diff
@@ -7,6 +7,13 @@
     Alike 3.0 License: http://creativecommons.org/licenses/by-sa/3.0/
    ****************************************************************************
 
+|
+
+
+* **Supported versions:**
+  (`3.2 <https://docs.pgrouting.org/3.2/en/pgr_makeConnected.html>`__)
```
Comment (cvvergara) [inline doc/components/pgr_makeConnected.rst:14]: Also add the latest
```
`Latest <https://docs.pgrouting.org/latest/en/pgr_makeConnected.html>`__)
```

**Review by cvvergara — CHANGES_REQUESTED** [review]

File: doc/components/pgr_makeConnected.rst (hunk 7 above)
Comment (MarPetra) [reply doc/components/pgr_makeConnected.rst:14]: the link does not work

Comment (cvvergara) [reply doc/components/pgr_makeConnected.rst:14]: Add the missing line
```
`Latest <https://docs.pgrouting.org/latest/en/pgr_makeConnected.html>`__
```
and on this file: doc/conf.py.in around line 236 we have a list of links that might not be working.
https://github.com/pgRouting/pgrouting/blob/develop/doc/conf.py.in#L326

```
'https://docs.pgrouting.org/lastes/en/pgr_makeConnected.html' ,
```
Use normal ' and put a comma at the end

**Comment by cvvergara** [comment]
## Reminder
When a link does not work
For example: <https://docs.pgrouting.org/latest/en/pgr_makeConnected.html
it is because the function is new and 3.2 is not yet the latest documentation

### How to solve:

Still add the missing line
```
`Latest <https://docs.pgrouting.org/latest/en/pgr_makeConnected.html>`__
```
On this file: doc/conf.py.in around line 236 we have a list of links that might not be working.
https://github.com/pgRouting/pgrouting/blob/develop/doc/conf.py.in#L326
```
'https://docs.pgrouting.org/lastes/en/pgr_makeConnected.html' ,
```
Use normal ' and put a comma at the end

File: doc/bdDijkstra/bdDijkstra-family.rst (hunk 1 above)
Comment (cvvergara) [reply doc/bdDijkstra/bdDijkstra-family.rst]: The file name is `bdDijkstra-family.rst` So after the `/en/` it should read `bdDijkstra-family.html`

File: doc/bdDijkstra/pgr_bdDijkstraCost.rst (hunk 2 above)
Comment (cvvergara) [reply doc/bdDijkstra/pgr_bdDijkstraCost.rst]: The file name is `pgr_bdDijkstraCost.rst` So after the `/en/` it should read `pgr_bdDijkstraCost.html`

File: doc/components/pgr_makeConnected.rst (hunk 7 above)
Comment (cvvergara) [reply doc/components/pgr_makeConnected.rst:14]: Latest goes before the 3.2

**Review by cvvergara — CHANGES_REQUESTED** [review]

**Comment by cvvergara** [comment]
## Reminder 2
When doing reminder 1, use the this branch, otherwise you will still get the link error missing
Do not open another PR

File: doc/coloring/coloring-family.rst [hunk 8]
```diff
@@ -7,6 +7,13 @@
     Alike 3.0 License: https://creativecommons.org/licenses/by-sa/3.0/
    ****************************************************************************
 
+|
+
+* **Supported versions:**
+  `Latest <https://docs.pgrouting.org/latest/en/coloring-family.html>`__
+  (`3.2 <https://docs.pgrouting.org/3.2/en/coloring-family.html>`__)
+
```
Comment (cvvergara) [inline doc/coloring/coloring-family.rst:15]: Add this link to the non existing links in conf.py.in **in this branch** do not do the change on the website, do it in your sublime
```
https://docs.pgrouting.org/latest/en/coloring-family.html
```

File: doc/coloring/pgr_bipartite.rst [hunk 9]
```diff
@@ -7,6 +7,12 @@
     Alike 3.0 License: http://creativecommons.org/licenses/by-sa/3.0/
    ****************************************************************************
 
+|
+
+* **Supported versions:**
+  `Latest <https://docs.pgrouting.org/latest/en/pgr_bipartite.html>`__
+  (`3.2 <https://docs.pgrouting.org/3.2/en/pgr_bipartite.html>`__)
```
Comment (cvvergara) [inline doc/coloring/pgr_bipartite.rst:14]: Add this link to the conf.py.in in this working branch
```
https://docs.pgrouting.org/latest/en/pgr_bipartite.html
```

File: doc/coloring/pgr_sequentialVertexColoring.rst [hunk 10]
```diff
@@ -7,6 +7,12 @@
     Alike 3.0 License: http://creativecommons.org/licenses/by-sa/3.0/
    ****************************************************************************
 
+|
+
+* **Supported versions:**
+  `Latest <https://docs.pgrouting.org/latest/en/pgr_sequentialVertexColoring.html>`__
+  (`3.2 <https://docs.pgrouting.org/3.2/en/pgr_sequentialVertexColoring.html>`__)
```
Comment (cvvergara) [inline doc/coloring/pgr_sequentialVertexColoring.rst:14]: Add this link to conf.py.in
```
https://docs.pgrouting.org/latest/en/pgr_sequentialVertexColoring.html
```

**Review by cvvergara — CHANGES_REQUESTED** [review]

File: doc/conf.py.in [hunk 11]
```diff
@@ -324,7 +324,19 @@ linkcheck_ignore = [
     # might not exist yet (we are generating it!)
     'https://docs.pgrouting.org/3.2/en/pgr_dijkstraNear.html',
     'https://docs.pgrouting.org/3.2/en/pgr_dijkstraNearCost.html',
+    'https://docs.pgrouting.org/3.2/en/pgr_randomSpanTree.html',
     'https://docs.pgrouting.org/lastes/en/pgr_makeConnected.html',
```
Comment (cvvergara) [inline doc/conf.py.in:328]: `lates` -> `latest`

File: doc/conf.py.in [hunk 12]
```diff
@@ -324,7 +324,19 @@ linkcheck_ignore = [
...
+    'https://docs.pgrouting.org/latest/en/coloring-family.html',
+    'https://docs.pgrouting.org/latest/en/pgr_bipartite.html',
+    'https://docs.pgrouting.org/latest/en/pgr_sequentialVertexColoring.html',
+    'https://docs.pgrouting.org/latest/en/pgr_dijkstraNear.html',
+    'https://docs.pgrouting.org/latest/en/pgr_dijkstraNearCost.html',
+    'https://docs.pgrouting.org/latest/en/pgr_lengauerTarjanDominatorTree.html',
+    'https://docs.pgrouting.org/latest/en/pgr_isPlanar.html',
+    'https://docs.pgrouting.org/latest/en/pgr_randomSpanTree.html',
+    'https://docs.pgrouting.org/latest/en/pgr_depthFirstSearch.html',
+    'https://docs.pgrouting.org/latest/en/traversal-family.html'
```
Comment (cvvergara) [inline doc/conf.py.in:338]: You are missing a `,` at the end

**Review by cvvergara — CHANGES_REQUESTED** [review]

File: doc/conf.py.in (hunk 12 above)
Comment (cvvergara) [reply doc/conf.py.in:338]: This missing comma is generating to many errors, please fix so I can continue with the review

File: doc/conf.py.in [hunk 13]
```diff
@@ -324,7 +324,20 @@ linkcheck_ignore = [
     # might not exist yet (we are generating it!)
     'https://docs.pgrouting.org/3.2/en/pgr_dijkstraNear.html',
     'https://docs.pgrouting.org/3.2/en/pgr_dijkstraNearCost.html',
-    'https://docs.pgrouting.org/lastes/en/pgr_makeConnected.html',
+    'https://docs.pgrouting.org/3.2/en/pgr_randomSpanTree.html',
+    'https://docs.pgrouting.org/lastest/en/pgr_makeConnected.html',
```
Comment (cvvergara) [inline doc/conf.py.in:328]: `lastest` -> `latest`

**Review by cvvergara — CHANGES_REQUESTED** [review]

File: doc/tsp/pgr_TSP.rst [hunk 14]
```diff
@@ -7,6 +7,22 @@
...
+  `3.0 <https://docs.pgrouting.org/3.0/en/pgr_TSP.html>`__
+* **Unsupported versions:**
+  `2.6 <https://docs.pgrouting.org/2.6/en/pgr_TSP.html>`__
+  `2.5 <https://docs.pgrouting.org/2.5/en/pgr_TSP.html>`__
+  `2.4 <https://docs.pgrouting.org/2.4/en/pgr_TSP.html>`__
+  `2.3 <https://docs.pgrouting.org/2.3/en/src/tsp/doc/pgr_tsp.html>`__
+  `2.2 <https://docs.pgrouting.org/2.2/en/src/tsp/doc/pgr_tsp.html>`__
+  `2.1 <https://docs.pgrouting.org/2.1/en/src/tsp/doc/index.html>`__
+  `2.0 <https://docs.pgrouting.org/2.0/en/src/tsp/doc/index.html>`__
+
```
Comment (cvvergara) [inline doc/tsp/pgr_TSP.rst:25]: make sure that there is no space or tab in line 25

**Review by cvvergara — CHANGES_REQUESTED** [review]

File: doc/trsp/pgr_trsp.rst [hunk 15]
```diff
@@ -7,7 +7,23 @@
...
+* **Unsupported versions:**
+  `2.6 <https://docs.pgrouting.org/2.6/en/pgr_trsp.html>`__
+  `2.5 <https://docs.pgrouting.org/2.5/en/pgr_trsp.html>`__
+  `2.4 <https://docs.pgrouting.org/2.4/en/pgr_trsp.html>`__
+  `2.3 <https://docs.pgrouting.org/2.3/en/src/trsp/doc/pgr_trsp.html>`__
+  `2.2 <https://docs.pgrouting.org/2.2/en/src/trsp/doc/pgr_trsp.html>`__
+  `2.1 <https://docs.pgrouting.org/2.1/en/src/trsp/doc/index.html>`__
+  `2.0 <https://docs.pgrouting.org/2.0/en/src/trsp/doc/index.html>`__
+
+  pgr_trsp - Turn Restriction Shortest Path (TRSP)
```
Comment (cvvergara) [inline doc/trsp/pgr_trsp.rst:26]: The title has 2 spaces on the left, remove them

**Comment by cvvergara** [comment]
My calculations indicate that these are missing the update
- [ ] pgRouting-concepts
- [ ] experimental
- [ ] index
- [ ] routingFunctions
- [ ] sampledata
- [ ] proposed
- [ ] pgRouting-installation
- [ ] pgr_alphaShape
- [ ] support
- [ ] topology-functions
- [ ] pgr_createTopology

**Review by cvvergara — APPROVED** [review]

**Pull request merged**